public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -116,41 +110,342 @@ const Policy: FC<PolicyComponentProps> = ({
+    ? sumBy(matchingAlertGroups, (group) => group.alerts.length)
+    : undefined;
+
+  // simplified routing permissions
+  const [isSupportedToSeeAutogeneratedChunk, isAllowedToSeeAutogeneratedChunk] = useAlertmanagerAbility(
+    AlertmanagerAction.ViewAutogeneratedPolicyTree
+  );
+  // collapsible policies variables
+  const isThisPolicyCollapsible = useShoudPolicyBeCollapsible(currentRoute);
+  const [isBranchOpen, toggleBranchOpen] = useToggle(false);
+  const renderChildPolicies = (isThisPolicyCollapsible && isBranchOpen) || !isThisPolicyCollapsible;
+
+  const groupBy = currentRoute.group_by;
+  const muteTimings = currentRoute.mute_time_intervals ?? [];
+
+  const timingOptions: TimingOptions = {
+    group_wait: currentRoute.group_wait,
+    group_interval: currentRoute.group_interval,
+    repeat_interval: currentRoute.repeat_interval,
+  };
+
   contactPointErrors.forEach((error) => {
     errors.push(error);
   });
 
-  const hasInheritedProperties = inheritedProperties && Object.keys(inheritedProperties).length > 0;
+  const isAutogeneratedPolicyRoot = isAutoGeneratedRootAndSimplifiedEnabled(currentRoute);
 
-  const childPolicies = currentRoute.routes ?? [];
+  // build the menu actions for our policy
+  const dropdownMenuActions: JSX.Element[] = useCreateDropdownMenuActions(
+    isAutoGenerated,
+    isDefaultPolicy,
+    provisioned,
+    onEditPolicy,
+    currentRoute,
+    toggleShowExportDrawer,
+    onDeletePolicy
+  );
+
+  // check if this policy should be visible. If it's autogenerated and the user is not allowed to see autogenerated
+  // policies then we should not show it. Same if the user is not supported to see autogenerated policies.
+  const hideCurrentPolicy =
+    isAutoGenerated && (!isAllowedToSeeAutogeneratedChunk || !isSupportedToSeeAutogeneratedChunk);
+
+  if (hideCurrentPolicy) {
+    return null;
+  }
+  const isRootPolicy = isDefaultPolicy || isAutogeneratedPolicyRoot;
+  // TODO dead branch detection, warnings for all sort of configs that won't work or will never be activated
+  return (
+    <>
+      <Stack direction="column" gap={1.5}>
+        <div
+          className={styles.policyWrapper(hasFocus)}
+          data-testid={isDefaultPolicy ? 'am-root-route-container' : 'am-route-container'}
+        >
+          {/* continueMatching and showMatchesAllLabelsWarning are mutually exclusive so the icons can't overlap */}
+          {continueMatching && <ContinueMatchingIndicator />}
+          {showMatchesAllLabelsWarning && <AllMatchesIndicator />}
+
+          <div className={styles.policyItemWrapper}>
+            <Stack direction="column" gap={1}>
+              {/* Matchers and actions */}
+              <div>
+                <Stack direction="row" alignItems="center" gap={1}>
+                  {isThisPolicyCollapsible && childPolicies.length > 0 && (
+                    <Icon name={isBranchOpen ? 'angle-down' : 'angle-right'} onClick={toggleBranchOpen} />
```

Comment:
use `IconButton`, more obvious that this is a clickable target and helps with a11y and keyboard navigation